src/panels/menu/help/HelpCheatSheet.tsx

```diff
@@ -1,23 +1,24 @@
 import React from 'react';
 import {ProjectSettings} from "../../../config/Variables";
-import {helpModal} from "../../../locale/help";
+import {LocaleHelp} from "../../../config/Locale";
```

Comment:
You know, if you add the following to `tsconfig.json` `compilerOptions` key:
```
"include": [
  "src"
],
```
You would be able to do this instead:
```
import {LocalHelp} from 'config/Locale'
```
Which helps with refactoring a lot :-)

Reply:
Hmm... I already have this key inserted, but it does not seem to work. If I am completely honest, though, I don't pay attention to imports even when heavily refactoring something, since Webstorm has nice functions that does everything for me, and I never had a problem with it. So I will try to look into it, but if you don't mind, I currently give it a low priority - but thank you nevertheless.